src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
The class Circle should not begin with an empty line after the class declaration. Please remove the redundant empty line.